package.json

```diff
@@ -70,7 +70,7 @@
     "graphql-request": "^6.1.0",
     "jsdom": "^24.1.0",
     "packemon": "3.3.1",
-    "permissionless": "^0.1.29",
+    "permissionless": "^0.2.10",
```

Comment:
can we remove this one as its a dependency too?

Reply:
removed